postr/instagram_postr.py

```diff
@@ -25,15 +32,31 @@ def __init__(self, user: Dict[str, Any]) -> None:
 class Instagram(ApiInterface):
     """ Wrapper for accessing the instagram API  """
 
+    # A workaround URL that retrieves a profile JSON without triggering a 403 error.
+    # URL before the username
+    PRE_PROFILE_URL = 'https://www.instagram.com/web/search/topsearch/?context=blended&query='
+    # URL after the username, which loads the profile JSON
+    POST_PROFILE_URL = '&rank_token=0.3953592318270893&count=1'
```

Comment:
move the 0.39.......93 to a config file?

Reply:
or extract it to its own constant with a description of the problem / link to issue where found